webapp/packages/core-blocks/src/ItemList/ItemListSearch.tsx

```diff
@@ -39,6 +39,15 @@ export const ItemListSearch: React.FC<IProps> = function ItemListSearch({ value,
     [value, onChange],
   );
 
+  const resetSearch = useCallback(() => {
+    if (value === undefined) {
+      setSearch('');
+    }
+    if (onChange) {
+      onChange('');
+    }
+  }, [value, onChange]);
+
```

Comment:
code duplication, you can change `changeHandler` so it will accept value instead of event, then you can call it in `onClick={() => changeHandler('')}`